pom.xml

```diff
@@ -141,13 +142,39 @@
         </dependency>
 
         <dependency>
```

Comment:
Added the following dependencies based upon https://blog.sagaoftherealms.net/?p=528